packs/data/feats.db/perfect-resistance.json

```
@@ -0,0 +1,57 @@
+{
+    "_id": "QN55ToyJIjxgPmhs",
+    "img": "systems/pf2e/icons/default-icons/feats.webp",
+    "name": "Perfect Resistance",
+    "system": {
+        "actionCategory": {
+            "value": ""
+        },
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "description": {
+            "value": "You gain resistance equal to half your level to all damage from effects with the appropriate trait for your School of Perfection: fire for Unblinking Flame, water for Unbreaking Waves, air for Unfolding Wind, and earth for Untwisting Iron. After using a Student of Perfection ki spell, the resistance increases to be equal to your level for 1 round.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Effect: Perfect Resistance]{Effect: Perfect Resistance}</p>"
+        },
+        "featType": {
+            "value": "archetype"
+        },
+        "level": {
+            "value": 8
+        },
+        "prerequisites": {
+            "value": [
+                {
+                    "value": "Student of Perfection Dedication"
+                }
+            ]
+        },
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.houseOfPerfection.energy",
```

```suggestion
                "path": "flags.pf2e.houseOfPerfection.element",
```
Let's change to element since that better reflects what it is we're storing.